src/CognitiveServices/CognitiveServices/ChangeLog.md

```diff
@@ -17,6 +17,9 @@
     * Overview of change #1
         - Additional information about change #1
 -->
+## Upcoming Release
```

Comment:
you have duplicate "upcoming release"

Reply:
Oh didn't know I should merge them. Updated!